ksqldb-functional-tests/src/test/resources/query-validation-tests/formats.json

```diff
@@ -2,6 +2,10 @@
   "tests": [
     {
       "name": "invalid value format - create source",
+      "config": ["true", "false"],
```

Comment:
Is this new "config" field meant for general purpose use? If so, we should add it to the docs: https://github.com/confluentinc/ksql/blob/master/ksqldb-functional-tests/README.md